app/app/models/user.py

```diff
@@ -1,5 +1,5 @@
 from sqlalchemy import Boolean, Column, Integer, String
-from sqlalchemy.dialects.postgresql import ENUM
+from sqlalchemy.dialects.mysql import ENUM
```

Comment:
won't we need an if-else check to see if we're using mysql or postgresql?

Reply:
Oh, forgot to update this.